crates/pallet-domains/src/staking.rs

```diff
@@ -1664,7 +1653,7 @@ pub(crate) mod tests {
             );
             assert_err!(
                 res,
-                Error::<Test>::Staking(crate::staking::Error::DuplicateOperatorSigningKey)
+                Error::<Test>::Staking(crate::staking::Error::InsufficientBalance)
```

Comment:
I'm not sure what to do here. I think this test should return ok, or another error. But increasing the free balance to 3000 or more doesn't fix this error.

Reply:
Huh, I must have been using an old test binary, this actually works.